api/V1/share_service.proto

```diff
@@ -0,0 +1,120 @@
+syntax = "proto3";
+
+package celestia.node.v1.share;
+
+option go_package = "github.com/celestiaorg/celestia-node/api/v1/share";
+
+import "common.proto";
+import "types.proto";
+import "proof.proto";
+
+message GetShareRequest {
+  uint64 height = 1;
+  uint32 row = 2;
+  uint32 col = 3;
+  optional celestia.node.v1.common.Options options = 4;
+}
```

Comment:
Will you also be able to reference a share based on an index (rather than rol and col). Some of the responses return the starting index of a blob.